v2/pink-sb/src/lib/selector/Base.svelte

```diff
@@ -13,13 +13,13 @@
             <label for={id}>
                 <Typography.Text variant="m-500">{label}</Typography.Text>
             </label>
-            <slot name="description">
-                {#if description}
-                    <Typography.Text variant="m-400" color="--fgcolor-neutral-tertiary"
-                        >{description}</Typography.Text
-                    >
-                {/if}
-            </slot>
+            {#if description}
+                <Typography.Text variant="m-400" color="--fgcolor-neutral-tertiary"
+                    >{description}</Typography.Text
+                >
+            {:else if $$slots.description}
+                <slot name="description"></slot>
+            {/if}
```

Comment:
Since we're passing the slot to the other inputs this might create some issues with the stack component